src/node/internal/process.d.ts

```diff
@@ -0,0 +1,7 @@
+export const versions: Record<string, string>;
+export function getEnvObject(): Record<string, string>;
+export function getBuiltinModule(id: string): object;
+export function processExitImpl(code: number): void;
+export const processPlatform: string;
```

Comment:
This file is already named "process", this prefixes seems to be redundant.

```suggestion
export function exitImpl(code: number): void;
export const platform: string;
```

Reply:
Added.